### PR TITLE
[9.0][ADD][account_invoice_refund_option] Create refund invoices directly

### PR DESCRIPTION
**Pull request by yajo**
This addon lets you create a refund out of nothing.

Includes workaround for https://github.com/odoo/odoo/pull/14187

@Tecnativa

### CODE REVIEW

File: account_invoice_refund_option/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Create Refund Invoice
+=====================
+
+This module extends the functionality of accounting invoices to support setting
+an invoice to be a refund and to allow you to create refunds directly, linked
```
Comment (pedrobaeza) [inline account_invoice_refund_option/README.rst]: s/linked to nothing/without the need of refunding an existing normal invoice.

File: account_invoice_refund_option/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Create Refund Invoice",
+    "summary": "Allows you to create a refund linked to nothing",
```
Comment (pedrobaeza) [inline account_invoice_refund_option/__openerp__.py]: `Allows you to create directly a refund without starting from an invoice`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Minor things + add Spanish translation instead of .empty file.

**Review by rafaelbn — APPROVED** [review]
Tested in runbot

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**